.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release:
```

Comment:
In other Spinnaker projects, we usually add the following to GitHub actions so that they don't run on contributors' forks:

```
if: startsWith(github.repository, 'spinnaker/')
```

Reply:
Added: [`ab67137` (#27)](https://github.com/spinnaker/kustomization-base/pull/27/commits/ab67137c639181544ad68a00f26da0cdd5413cc4)